src/main/java/org/trebol/jpa/services/conversion/PeopleConverterJpaServiceImpl.java

```diff
@@ -72,11 +73,11 @@ public Person applyChangesToExistingEntity(PersonPojo source, Person existing) t
 
     // phones may be empty, but not null
     String phone1 = source.getPhone1();
-    if (phone1 != null) {
-      if (!target.getPhone1().equals(phone1)) {
+
+      if ( !StringUtils.equals(target.getphone1(),phone1)) {
```

Comment:
Please remove the extra space :)

Reply:
in line 77

it should be 

`if (!StringUtils.equals(target.getphone1(),phone1))`